homework34/src/main/java/com/vadim/tkach/lesson34/Stack.java

```diff
@@ -0,0 +1,63 @@
+package com.vadim.tkach.lesson34;
+
+import java.util.Optional;
+
+public class Stack implements Stackable {
+    private final int maxSize;
+    private final int[] stackArray;
+    private int top;
+
+    public Stack(int max) {
+        this.maxSize = max;
+        stackArray = new int[maxSize];
+        top = -1;
+    }
+
+    @Override
+    public void addElementToStack(int element) throws StackIsFullException {
+        if (isFull()) {
+            throw new StackIsFullException();
+        } else {
+            stackArray[++top] = element;
+            System.out.println("Element " + element + " be addet to stack");
+        }
```

Comment:
`else` прописувати не обовязково. Якщо програма заходить в середину `if (isFull()) {` то буде викидуватись помилка і програма зупиниться (далі не буде виконватися).
```suggestion
        if (isFull()) {
            throw new StackIsFullException();
        }
        
        stackArray[++top] = element;
        System.out.println("Element " + element + " be addet to stack");
```